features/index.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+   As a User|
+   In order to navigate index.erb
+   I want to see a menu
+
+   Backgroud:
+   Given that this is the first page user arives to, there are no previous page.
```

Comment:
This step is unnecessarily complicated. It also doesn't need to be here at all, because we are doing `When I visit the site` in the first step in the scenario.